### ISSUE TITLE
differential_abundance.py unit test failure

### ISSUE THREAD

**Issue by jairideout** [biocore/qiime#1981]
The following differential_abundance.py unit test is failing when Jenkins runs the nightly tests:

```
Testing /mnt/workspace/QIIME/qiime/tests/test_differential_abundance.py:

test_DESeq2_nbinom_format (__main__.RDifferentialAbundanceTests) ... ok
test_check_mapping_file_category (__main__.RDifferentialAbundanceTests) ... ok
test_metagenomeSeq_fitZIG_format (__main__.RDifferentialAbundanceTests) ... FAIL

======================================================================
FAIL: test_metagenomeSeq_fitZIG_format (__main__.RDifferentialAbundanceTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/workspace/QIIME/qiime/tests/test_differential_abundance.py", line 142, in test_metagenomeSeq_fitZIG_format
    self.assertAlmostEqual(af_e, ef_e)
AssertionError: 0.000958993450178425 != 0.000370569 within 7 places

----------------------------------------------------------------------
Ran 3 tests in 102.986s

FAILED (failures=1)
```

@sowe9385 can you please look into this? This test started failing around 5 days ago, maybe it's related to a new version of metagenomeSeq?

**Comment by sowe9385** [comment]
Yes, all Bioconductor packages were updated in the last few weeks, and it
looks like it is related.  I'll do a new PR?  Thanks.

**Comment by jairideout** [comment]
That'd be fantastic, thank you! We should also note what version(s) of these new R packages have been tested with QIIME. The [install docs](http://qiime.org/install/install.html) are probably a good place for this info.

**Comment by gregcaporaso** [comment]
@sowe9385, do you have an ETA on this? We're not able to use jenkins effectively right now b/c ``test_differential_abundance.py`` is failing.

**Comment by gregcaporaso** [comment]
This is fixed in #1986.
